src/transformers/models/auto/modeling_auto.py

```diff
@@ -577,6 +578,7 @@
 MODEL_FOR_SEQUENCE_CLASSIFICATION_MAPPING_NAMES = OrderedDict(
     [
         # Model for Sequence Classification mapping
+        ("graphormer", "GraphormerForGraphClassification"),
```

Comment:
Is this really going to follow the same API as traditional text classification models?

Reply:
Ultimately, it would be great to have a Graph Classification pipeline, can I just create a new `MODEL_FOR_GRAPH_CLASSIFICATION_...` dict here?

Reply:
When there is a pipeline to go with it. No need to create the auto class right now.